birdhouse/config/thredds/docker-compose-extra.yml

```diff
@@ -19,8 +18,8 @@ services:
       PAVICS_FQDN_PUBLIC: $PAVICS_FQDN_PUBLIC
     volumes:
       - thredds_persistence:/usr/local/tomcat/content/thredds
-      - ${DATA_PERSIST_ROOT}/datasets:/pavics-data
-      - ${DATA_PERSIST_ROOT}/ncml:/pavics-ncml
+      - ${THREDDS_SERVICE_DATA_LOCATION_ON_HOST}:${THREDDS_SERVICE_DATA_LOCATION_ON_CONTAINER}
+      - ${THREDDS_DATASET_LOCATION_ON_HOST}:${THREDDS_DATASET_LOCATION_ON_CONTAINER}
       - wps_outputs:/pavics-data/wps_outputs
```

Comment:
`/pavics-data` here to replace by the proper var.  Do a full repo search for the string `/pavics-data` and `/pavics-ncml` to be sure.

Reply:
Good catch! Thanks!